translations/messages.fr.yml

```diff
@@ -16,6 +16,8 @@ approved: Approuvé
 #
 Invalid credentials.: L'adresse e-mail et le mot de passe que vous avez saisis ne correspondent pas.
 Max attempts reached.: Vous avez effectué 5 tentatives de connexion erronées. Veuillez attendre 1 minute avant de réessayer.
+Account have to be validated ! Please Check your mailbox.: Pour vous connecter vous devez confirmer votre adhésion. Si vous n'avez pas reçu le mail de validation, vous pouvez <a href="/renvoyer-validation">cliquer ici</a> pour le recevoir à nouveau.
+Account is disabled.: Votre compte à été désactivé.
```

Comment:
Same here

Reply:
Not possible for this one , I use an vendor exception class with this message